src/scripts/tray.js

```diff
@@ -24,6 +24,17 @@ const icons = {
 
 const _iconTray = path.join(__dirname, 'images', icons[process.platform].dir, icons[process.platform].icon || 'icon-tray.png');
 const _iconTrayAlert = path.join(__dirname, 'images', icons[process.platform].dir, icons[process.platform].iconAlert || 'icon-tray-alert.png');
+const _iconTrayTitleDot = path.join(__dirname, 'images', icons[process.platform].dir, icons[process.platform].iconTrayTitleDot || 'title/icon-tray-title-dot.png');
```

Comment:
Is there a difference between the linux and windows icons? If not you might as well put them all in a parent folder.